examples/meta-model/README.md

```diff
@@ -0,0 +1,17 @@
+# Meta Model
```

Comment:
Same as https://github.com/asyncapi/modelina/pull/530/files#r808839189

In fact, you are referring to it as `meta model` in the line below.

Reply:
Changed 😄